### ISSUE TITLE
What is the validation set you use for supervised finetuning?

### ISSUE THREAD

**Issue by raghavlite** [McGill-NLP/llm2vec#41]
What is the validation set you use for supervised finetuning?
is it the 15 task subset?

**Comment by vaibhavad** [comment]
Thanks for your interest in our work. As MTEB does not have a validation set, we arbitrarily decided to run every training procedure - MNTP, SimCSE, supervised - for 1000 steps. The number of steps as a hyperparameter has not been optimized on any dev set.

**Comment by raghavlite** [comment]
thanks
